pandas/tests/frame/test_constructors.py

```diff
@@ -501,9 +501,11 @@ def test_constructor_dict_of_tuples(self):
         tm.assert_frame_equal(result, expected, check_dtype=False)
 
     def test_constructor_dict_multiindex(self):
-        check = lambda result, expected: tm.assert_frame_equal(
```

Comment:
can you parameterize this test?

Reply:
Not here, but I can avoid fixing the lambda